projects/plugins/jetpack/tests/php/modules/masterbar/test-class-atomic-admin-menu.php

```diff
@@ -167,8 +173,11 @@ public function test_add_new_site_link() {
 	 * Tests add_site_card_menu
 	 *
 	 * @covers ::add_site_card_menu
+	 *
+	 * @expectedDeprecated Automattic\Jetpack\Dashboard_Customizations\Atomic_Admin_Menu::add_site_card_menu
 	 */
 	public function test_add_site_card_menu() {
+		$this->markTestSkipped( 'Temporary skipping' );
```

Comment:
Temporarily marking as skipped till I figure out how to best handle: `plugins_url( 'modules/masterbar/admin-menu/globe-icon.svg', JETPACK__PLUGIN_FILE ),`

Reply:
Does that mean you'd be looking to handle that in a later PR, or later in this PR? If in a later PR it looks like adding `// @phan-suppress-next-line PhanPluginUnreachableCode` would be needed straight after the line where you've marked the test as skipped to prevent the current phan error.

Reply:
Ideally I'd like to handle this in this PR, phan failing serves as a reminder for me so I don't forget. Since you already tested the functionality I'll ask Brad for a review in case he has any suggestions for this part. If not, I could just update the test to cover the deprecation part only.

Reply:
The `->assertEquals()` in this test has the `$expected` and `$actual` options backwards. That confused me at first.

Can you not just update it to `plugins_url( 'jetpack_vendor/automattic/jetpack-masterbar/src/admin-menu/globe-icon.svg', JETPACK__PLUGIN_FILE )`?

Reply:
> The ->assertEquals() in this test has the $expected and $actual options backwards. That confused me at first.

Same, that's why I had it [fixed in the package](https://github.com/Automattic/jetpack/blob/trunk/projects/packages/masterbar/tests/php/test-class-atomic-admin-menu.php#L206)

> Can you not just update it to plugins_url( 'jetpack_vendor/automattic/jetpack-masterbar/src/admin-menu/globe-icon.svg', JETPACK__PLUGIN_FILE )?

Unfortunately not, locally I get:
```
1) Test_Atomic_Admin_Menu::test_add_site_card_menu
Failed asserting that two arrays are equal.
--- Expected
+++ Actual
@@ @@
     3 => 'site-card'
     4 => 'menu-top toplevel_page_http:/...le.org'
     5 => 'toplevel_page_http://example.org'
-    6 => 'http://example.org/wp-content/plugins/usr/local/src/jetpack-monorepo/projects/packages/masterbar/src/admin-menu/globe-icon.svg'
+    6 => 'http://example.org/wp-content/plugins/usr/local/src/jetpack-monorepo/projects/plugins/jetpack/jetpack_vendor/automattic/jetpack-masterbar/src/admin-menu/globe-icon.svg'
 )
 ```

Reply:
For now I just updated it the unit test leaving only the deprecation expectation via https://github.com/Automattic/jetpack/pull/38109/commits/b2b575a0527820b2350deb5cefa2e241540c51dc

Reply:
Hmm. In CI it should work because there aren't symlinks involved. In the `jetpack docker` dev environment it should probably also work thanks to `tools/docker/mu-plugins/fix-monorepo-plugins-url.php` (although I could see it picking the "wrong" monorepo plugin for some cases if the package is used in multiple places).

Anyway, the "for now" seems fine to me. 👍